web/app/js/components/BaseTable.jsx

```diff
@@ -0,0 +1,32 @@
+import { Table } from 'antd';
+
+export default class BaseTable extends Table {
+  constructor(props) {
+    super(props);
+    Table.prototype.toggleSortOrder = this.toggleSortOrder;
+  }
+
+  toggleSortOrder(order, column) {
+    let { sortColumn, sortOrder } = this.state;
+    let isSortColumn = this.isSortColumn(column);
+    sortOrder = order;
+    sortColumn = column;
+
+    const newState = {
```

Comment:
I think lines 10-18 are a little repetitive. Given that we're not using a lot of the overwriting in the original function, 
I think this can be simplified by eliminating the intermediate variables and doing
```
    const newState = {
      sortOrder: order,
      sortColumn: column,
    };
```

Reply:
That's true. I tried to leave as much of the original method "as-is", but this was too much as is. Changed it 🙂